.github/PULL_REQUEST_TEMPLATE

```diff
@@ -4,12 +4,12 @@ Congratulations! You're submitting your assignment! These comprehension question
 ## Comprehension Questions
 | Question | Answer    |
 | :------------- | :------------- |
-| How did your team break up the work to be done? |    |
-| How did your team utilize git to collaborate? |    |
-| What did your group do to try to keep your code DRY while many people collaborated on it?  |    |
-| What was a technical challenge that you faced as a group?  |    |
-| What was a team/personal challenge that you faced as a group?  |    |
-| What could your team have done better?  |    |
-| What was your application's ERD? (include a link) |    |
-| What is your Trello URL? |        |
-| What is the Heroku URL of your deployed application? |        |
+| How did your team break up the work to be done? | We started with assigning people to Trello cards.  the models and controllers for testing and creating the methods. We set up the tasks in waves on Trello and as we completed the tasks, we moved on to different tasks in the waves. |
```

Comment:
Also, these questions shouldn't be answered like this within this file!

github uses this file to automatically populate the text box for the description when you open a new Pull Request. Just edit that PR description. I went ahead and copied and pasted this into your PR for you :P